Script.md

```diff
@@ -0,0 +1,27 @@
+## Introducing Lodgel – the property management application that streamlines the rental process.
+Lodgel removes the hassle of ensuring your properties are ready for guests by coordinating assistants
+between properties and customizing checklists with tasks based on the needs of your guests. And if your guests
+have a special request? No problem. The checklists can be updated anytime something else is needed, and you can 
+check in on your assistants to ensure the job is done on time. We even integrated Stripe - one of the world's leading
+payment processors, so you can get paid as soon as your guests check out.
+
+Users register for an account, and right away, they can start adding properties to their list.
+To ensure each guest is taken care of, checklists can be made that detail tasks that need to be taken care of
+before, during, and after each visit. 
+
+Assistants are an essential element of every property management business. Each property has a turnover assistant 
+assigned to take care of it by default, but they can be moved around if necessary. Each assistant 
+has a map attached to their profile, which conveniently shows their location relative to the property they
+maintain. They can also select other properties that they are able to help take care of, saving you time 
+and effort in finding someone to cover another assistant in the event they are unavailable.
+
+You can also keep tabs on your guest throughout their entire rental experience. After putting in a reservation,
+you can keep your assistants up to date with your guests needs by adding items to their checklists. Since your 

```

Comment:
Checklists can't be altered retro-actively.

Reply:
✔️